src/index.js

```diff
@@ -18,7 +18,13 @@ import injectJSXWrapper from "./helpers/runtime/jsx-wrapper";
 
 export default function ({ Plugin, types: t }) {
   return new Plugin("incremental-dom", { visitor : {
-    Program: setupInjector,
+    Program: {
+      enter: [setupInjector, setupHoists],
```

Comment:
It [normalizes](https://github.com/babel/babel/blob/369b9fdc4bf541933f24184a7e8ad7ec0776d600/packages/babel/src/transformation/plugin.js#L83-L86) visitors [into an array](https://github.com/babel/babel/blob/369b9fdc4bf541933f24184a7e8ad7ec0776d600/packages/babel/src/traversal/visitors.js#L10). :fireworks: